pvlib/pvsystem.py

```diff
@@ -1064,6 +1064,144 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
     return iam
 
 
+def iam_martin_ruiz(θ:'degrees', a_r=0.16):
```

Comment:
current naming pattern is `[model]iam`, but I prefer this. In the long run (*maybe* 0.7) I'd prefer a new `iam` module, but we can set that aside for the moment.

Reply:
I think unicode in function definitions is too big a leap for now. Maybe in the future. I think I previously stated my support for using it in examples/documentation.

I'm not familiar with the `:'unit'` syntax and didn't easily find a reference. link?

Reply:
Just testing the water with all three of these, and looking for reactions.

- names are intentionally breaking with the existing pattern
- unicode--the future could be now or later
- the annotation 'degrees' I find nice for interactive work, but tend to use it sparingly; here is some info: [https://www.geeksforgeeks.org/function-annotations-python/](url)